app/components/RegionDropdown/RegionDropdown.tsx

```diff
@@ -40,8 +41,6 @@ export const RegionDropdown = ({
         })}
       </select>
       <div className={styles["dropdown-caret"]}>
-        {/* <span className={styles["caret-left"]} />
```

Comment:
Imo we may as well delete the CSS for this as well

Reply:
Already gone ;)